### ISSUE TITLE
Builds successfully, but doesn't seem to work

### ISSUE THREAD

**Issue by nikolov-tmw** [markmandel/psmouse-focaltech-dkms#2]
Thank you for your work - it's highly appreciated! 

So, I tried installing the .deb package as well as running the `install.sh` script - both lead to the module installing and building properly(no errors occurring), but when I reboot the touchpad is still in emulation mode(no gestures). 

Is there any debug info that I can provide to help you with debugging this(does the dkms module work for you)?

I've tried building the module against two mainline kernels( 3.17.1-031701 and 3.17.0-031700rc-6 ), as well as the latest kernel shipped with 14.10( 3.16.0-30 ), but the result is the same in both cases. 

Thanks, 
Nikola

**Comment by markmandel** [comment]
Yeah, I never managed to get it to work, or work out why it didn't work either. I haven't tried the PR.

Your best bet would be to grab a kernel from:
https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1372609
